From 4e872963a318c773dd5fa2a27cbb24c911a9898f Mon Sep 17 00:00:00 2001
From: Lei Maohui <leimaohui@fujitsu.com>
Date: Thu, 13 Mar 2025 04:07:03 +0000
Subject: [PATCH] There is a conflict of kea-dhcp6.conf between kea and lib32-kea:

file /etc/kea/kea-dhcp6.conf conflicts between attempted installs of kea-2.6.1-r0.aarch64 and lib32-kea-2.6.1-r0.armv7ahf_neon

Because they are all commented out, replace the expanded libdir path with '$libdir' in the config files to avoid conflict.

Signed-off-by: Lei Maohui <leimaohui@fujitsu.com>
---
 src/bin/keactrl/kea-dhcp6.conf.pre | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/bin/keactrl/kea-dhcp6.conf.pre b/src/bin/keactrl/kea-dhcp6.conf.pre
index 271021b..5b85854 100644
--- a/src/bin/keactrl/kea-dhcp6.conf.pre
+++ b/src/bin/keactrl/kea-dhcp6.conf.pre
@@ -201,7 +201,7 @@
     //       // of all devices serviced by Kea, including their identifiers
     //       // (like MAC address), their location in the network, times
     //       // when they were active etc.
-    //       "library": "@libdir@/kea/hooks/libdhcp_legal_log.so",
+    //       "library": "$libdir/kea/hooks/libdhcp_legal_log.so",
     //       "parameters": {
     //           "path": "/var/lib/kea",
     //           "base-name": "kea-forensic6"
@@ -218,7 +218,7 @@
     //       // of specific options or perhaps even a combination of several
     //       // options and fields to uniquely identify a client. Those scenarios
     //       // are addressed by the Flexible Identifiers hook application.
-    //       "library": "@libdir@/kea/hooks/libdhcp_flex_id.so",
+    //       "library": "$libdir/kea/hooks/libdhcp_flex_id.so",
     //       "parameters": {
     //           "identifier-expression": "relay6[0].option[37].hex"
     //       }
-- 
2.43.0

